src/app/infra/repositories/sensors_records_repository.py

```diff
@@ -47,3 +47,24 @@ def __get_sensors_record(self, row: Dict) -> SensorsRecord:
             water_volume=row["water_volume"],
             created_at=row["created_at"],
         )
+    def get_last_date(self):
```

Comment:
It is easier to just return  the last sensors record as it already has the datetime.